src/test/java/Safron/NewBalanceUa/Tests/SignIn.java

```diff
@@ -0,0 +1,32 @@
+package Safron.NewBalanceUa.Tests;
+
+import BasePageAndTestInit.TestInit;
+import Safron.NewBalanceUa.Pages.NewBalanceHeaderViewElements;
+import Safron.NewBalanceUa.Pages.NewBalanceSignInViewElements;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SignIn extends TestInit {
+
+    @DataProvider(name = "user info")
+    public Object[][] dpMethod() {
+        return new Object[][]{
+                {"elsafronitaz@gmail.com", "E8crmWqTtYU2GD7"}};
+    }
+
+    @Test(dataProvider = "user info")
+    public void signIn(String email, String password) {
+        openUrl("https://newbalance.ua/");
```

Comment:
move openUrl() method on the start of the test (line 24)

Reply:
fix